council/council.html

```diff
@@ -48,7 +48,7 @@ <h4>Nearby</h4>
       <p>This document supplements the Process Document by providing best practices to resolve and decide Formal Objections.</p>
 
       <p>In this document, <dfn id="resolving">resolving</dfn> a Formal Objection means finding a solution that has no objections. <dfn id="deciding">Deciding</dfn> means 
-        giving a yes/no answer to the question of whether an objection should be <dfn id="overrule">overruled</dfn> (i.e., no) or <dfn id="sustained">sustained</dfn> (i.e., yes).
+        giving a yes/no answer to the question of whether an objection should be <dfn id="overrule">overruled</dfn> (i.e., no) or <dfn id="sustained">sustained</dfn> (i.e., yes); <a href="#council">Council</a> refers to the ephemeral W3C Council created to make a <a href="deciding">decision</a> on one or more Formal Objections. Councils comprising different members may exist at the same time to handle distinct decisions.
```

Comment:
Punctuation is inconsistent here. Choose whether you want a period or semicolon between definitions. :)